ark/segmentation/marker_quantification_test.py

```diff
@@ -61,12 +67,13 @@ def test_compute_marker_counts():
     assert np.array_equal(segmentation_output.loc['whole_cell', 1:, 'cell_size'],
                           segmentation_output.loc['whole_cell', 1:, 'area'])
 
-    # test whole_cell and nuclear compartments with same data
```

Comment:
I think keeping this comment, or some similar comment indicating what is going on here, is worthwhile

Reply:
Oops, I think that line was just an innocent bystander 😬